docs/continuous-delivery/low-code-solutions/low-code-solutions.md

```diff
@@ -0,0 +1,37 @@
+# Continuous delivery on low code solutions
+
+Low code, no code platforms have taken a spot in a wide variety of Business Solutions involving process automation, AI models, Bots, Business Applications and Business Intelligence. The scenarios enabled by these platforms are constantly evolving and opening a spot for productive roles. This has been exactly the reason why bringing more professional tools to their development has became necessary such as controlled and automated delivery.
+
+In the case of Power Platform products, the adoption of a CI/CD process may increase the development complexity to a solution oriented to [Citizen Developers](https://www.gartner.com/en/information-technology/glossary/citizen-developer) but, on the other hand, it makes the development process more scalable and capable of dealing with new features and bug corrections in a faster way.
+
+## Environments in Power Platform Solutions
+
+Environments are spaces where Power Platform Solutions exists. They store, manage and share everything related to the solution like data, apps, chatbots, flows and models. They also serve as containers to separate apps that might have different roles, security requirements or just target audiences. They can be used to create different stages of the solution development process, the expected model of working with environments in a CI/CD process will be as the following image suggests.
+
+![image](../images/environments.png)
+
+### Environments considerations
+
+Whenever an environment has been created, its resources can be only accessed by users within the same tenant. When you create an app in an environment that app can only interact with data sources that are also deployed in that same environment, this includes connections, flows and Dataverse databases. This is an important consideration when dealing with a CD process.
+
+## Deployment strategy
+
+With three environments already created to represent the stages of the deployment, the goal now is to automate the deployment from one environment to another. Each environment will require the creation of its own solution (data included).
```

Comment:
do this "... its own solution (data included)." means that the data is also required in the environment? If so it should be more descriptive:  "Each environment will require the creation of its own solution: business logic and data."